src/components/Service/index.js

```diff
@@ -1,41 +1,89 @@
 import React from "react";
 import PropTypes from "prop-types";
+import Link from "gatsby-link";
 
 import Typography from "root/components/Typography";
 import capitalize from "lodash/capitalize";
-import asterisk from "root/assets/images/asterisk.svg";
+import Jelly from "jelly.js";
 
 import "./index.css";
 
-const Service = ({ serviceName, image, specificServices }) => (
-  <div styleName="services">
-    <div styleName="service-type">
-      <img styleName="asterisk" src={asterisk} alt="asterisk" />
-      <div styleName="type">
-        <Typography variant="h3" color="dark-blue">
-          {capitalize(serviceName)}
-        </Typography>
-      </div>
-    </div>
-    <div styleName="service-description">
-      <div>
-        <img src={image} alt={serviceName} />
-      </div>
-      <div styleName="description">
-        {specificServices.map(specificService => (
-          <Typography variant="body" color="dark-blue" key={specificService}>
-            {specificService}
-          </Typography>
-        ))}
-      </div>
-    </div>
-  </div>
-);
+export default class Service extends React.Component {
+  static propTypes = {
+    serviceName: PropTypes.string.isRequired,
+    image: PropTypes.string.isRequired,
+    specificServices: PropTypes.arrayOf(PropTypes.string).isRequired,
+  };
+
+  componentDidMount = () => {
+    const { serviceName } = this.props;
+    const options = {
+      paths: `#circle-path-${serviceName}`,
+      color: "#5764F7",
+      intensity: 0.7,
+      pointsNumber: 8,
+      fastness: 1 / 40,
+      mouseIncidence: 20,
+      maxDistance: 40,
+    };
 
-Service.propTypes = {
-  serviceName: PropTypes.string.isRequired,
-  image: PropTypes.string.isRequired,
-  specificServices: PropTypes.arrayOf(PropTypes.string).isRequired,
-};
+    this[`jelly-${serviceName}`] = new Jelly(
```

Comment:
Problem is here. You need to do this on the `componentDidMount` otherwise the ServerSideRender mechanism of Gatsby will try to create the animation!

Reply:
But I am doing on `componentDidMount`.

Reply:
But I'll check if `this[jelly-${serviceName}]` exists

Reply:
The `new Jelly` is attaching to the `document` object, and that doesn't exist during the `build` process. You need to do this on `state` or create a `jelly icon` component that handles this.